### PR TITLE
Himmelstein's proofread

### PR DESCRIPTION
**Pull request by dhimmel**
- [x] update keywords
- [x] CSL updates in https://github.com/greenelab/meta-review/pull/203#issuecomment-481797833

### CODE REVIEW

File: content/02.main-text.md
Comment (dhimmel) [inline content/02.main-text.md]: https://ieeexplore.ieee.org/document/7884606 includes links to both the geekfeminism wiki and contributor covenant and is a more comprehensive discussion of open source COCs. I was considering keeping the geekfeminism wiki references as a link to "have used", but the wiki hasn't been updated since 2017, so I don't think it's adding anything beyond the current citation.

File: content/01.abstract.md
Comment (dhimmel) [inline content/01.abstract.md]: @agitter @slochower @cgreene want to start reviewing my updated abstract?

Is longer than before, with the goal of providing a more concrete description of what the paper is about. I measured 208 words ([PLOS CB](https://journals.plos.org/ploscompbiol/s/submission-guidelines) limit is 300).

**Comment by agitter** [comment]
@dhimmel this is out of scope of what you have edited so far, but I'm viewing this PR as our final pass through the manuscript.  I noticed that Distill uses a [reviewer bot](https://github.com/distillpub/post--visual-exploration-gaussian-processes/issues/1) to post reviews as GitHub issues an preserve anonymity.  That is exactly what we proposed here:
> Alternatively, a reviewer can post feedback with a pseudonymous GitHub account or have a trusted third party such as a journal editor post their comments anonymously.

We could considering citing Distill or linking to one of their reviews as an example of that idea.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:445]: I'd like to keep the contributor covenant reference as a way of using a relevant primary source and giving credit to its creator.  I'm open to removing the geekfeminism link and replacing it with the IEEE paper, which appears to have a similar goal but with more data.

File: content/01.abstract.md
Comment (agitter) [reply content/01.abstract.md]: Cross referencing #115.  We could also add this pull request to the response letter, but then we can't finalize the response letter link in this pull request.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: `a new software tool` could be `new software`

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md:5]: The idea that `automation can catch errors` is important and isn't as explicit in the new abstract.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: I'm not sure that `continuous` is a common enough idea to stand on its own here.  Could we move this to be part of the next sentence?  Something like?
> When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page, providing continuous publication.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: `persistent identifiers` is also not a familiar term.  Can we add an example or two?

We also have `rendered` in consecutive sentences.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: This is a long sentence but I don't see how to break it up.

**Review by agitter — COMMENTED** [review]
I made quick pass through and may have more comments later.  Overall, it is a much better abstract that actually tells readers what the paper is about.

**Review by agitter — COMMENTED** [review]
_Edit - double post and delete isn't working_

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: We probably should just switch to using a tag that will correspond to the GitHub release. Perhaps `v3.0`?

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md:445]: In https://github.com/greenelab/meta-review/pull/211/commits/c40e2fa444adeaded76d53fdcfad362dfb475175, I readded Contributor Covenant as a link rather than a citation. It doesn't make much sense IMO to cite a homepage that is likely to change over time. I considered citing version 4.1 of the covenant, but that seemed too narrow (also didn't include a date or author list), so I think it just makes sense to link to the homepage.

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: How about switching to a colon:

```
Publication with Manubot is continuous:
When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
```

I like the flow of having the clauses separate, and the colon should make the explanatory nature of the second sentence more clear.

Capitalization style info for this situation at https://www.grammarly.com/blog/capitalization-after-colons/

Comment (slochower) [reply content/01.abstract.md]: This is really nitpicky, but wouldn't a semicolon be the appropriate punctuation in that scenario?

File: content/01.abstract.md
Comment (slochower) [reply content/01.abstract.md]: `rendered` → `displayed` should be fine. And consider `specified` → `user specified` so that it's clear that it is customizable.

File: content/01.abstract.md
Comment (slochower) [reply content/01.abstract.md]: I think this is fine, but note that only half the authors of this paper were associated with Deep Review. This could be rephrased by eliminating "we" and just saying `an issue that was faced in Manubot's first use case...`

File: content/01.abstract.md
Comment (slochower) [reply content/01.abstract.md]: Added a few comments; LGTM overall.

File: content/01.abstract.md
Comment (slochower) [reply content/01.abstract.md:5]: I think this can go between the sentence beginning `Manubot adopts...` and `With Manubot...`.

**Review by slochower — APPROVED** [review]

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: From https://writingcenter.unc.edu/tips-and-tools/semi-colons-colons-and-dashes/

> Colons follow independent clauses (clauses that could stand alone as sentences) and can be used to present an explanation, draw attention to something, or join ideas together.

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md:5]: Do we think this sentence is now done justice by way of:

> A cloud service automatically evaluates proposed changes to catch errors.

along with 

> precisely track changes over time

and 

> making it transparent

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: I think that saying we doesn't necessarily mean everyone and provides the context that some of us are writing based on our experience with Deep Review rather than as outside parties. I also like the active voice. @agitter do you think this is issue?

File: content/01.abstract.md
Comment (agitter) [reply content/01.abstract.md:5]: Yes

File: content/01.abstract.md
Comment (agitter) [reply content/01.abstract.md]: A colon follows one independent clause, but a semicolon is better between two independent clauses.  Either way, the punctuation makes for a complex sentence in the abstract.

Can we keep separate sentences but make it active:
> Manubot provides continuous publication.
When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.

A downside is that many sentences will start with `Manubot ...`

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: `reproducible`

File: content/01.abstract.md
Comment (agitter) [reply content/01.abstract.md]: We could completely remove the Deep Review as a solution.  I like the abstract focusing almost entirely on Manubot because that is the purpose of the manuscript.  We are writing based on our experience with prior open collaborative projects, but the introduction makes that explicit.

I also like closing on the `Manubot modernizes publishing...` sentence.  If these two sentences (`Yet, fully open...` and `We discuss solutions...`) do stay, can we move them earlier?  They might fit after `However, traditional research...` with some revision.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md]: If you take my suggestion below to remove the last two sentences of the abstract, this could change to
> We present new software named Manubot, available at <https://manubot.org>, to address the challenges of open scholarly writing.

File: content/01.abstract.md
Comment (agitter) [inline content/01.abstract.md:9]: We've discussed the continuous publishing sentence below.  We could also merge it here and leave the `When a manuscript's...` sentence as is:
> A cloud service automatically evaluates proposed changes to catch errors and provide continuous publishing.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:132]: The tag works well here.  In the `Continuous publication` section we also have a link to a versioned URL.  Should update that to show a draft that includes all the authors?

File: content/metadata.yaml
Comment (agitter) [inline content/metadata.yaml:4]: These look good.  I'll see how many the submission system allows.

File: content/response-to-reviewers.md
Comment (agitter) [inline content/response-to-reviewers.md]: Noting that we may need to revise this after finalizing the abstract.

**Review by agitter — COMMENTED** [review]
A few more comments, primarily focused on the abstract.

File: content/01.abstract.md
Comment (vincerubinetti) [reply content/01.abstract.md]: I think we might be overthinking it here. The sentence as written with the colon looks totally fine and normal to me. Regardless of what the official english/websters dictionary grammar specifies, I've seen sentences written like that with a colon all the time.

Comment (dhimmel) [reply content/01.abstract.md]: > A colon follows one independent clause, but a semicolon is better between two independent clauses.

@agitter I think you may not be aware of this usage of a colon where the colon indicates that the second independent clause explains the first. See the [grammergirl examples](https://www.grammarly.com/blog/capitalization-after-colons/):

> It’s been snowing for three days straight: The roads around here aren’t very safe for driving.

> Maggie wears a brimmed cap at all times: Strong light often gives her a headache. She also likes the way it looks.

> Either way, the punctuation makes for a complex sentence in the abstract.

I like the colon because I think it maintains most of the separation and succinctness of the current construction. In other words, I don't really view the colon as creating a complex sentence. To the contrary, I think it helps retain simplicity while indicating the second sentence explains continuous publication.

Regarding "Manubot provides continuous publication.", I think it's less persuasive with a worse flow, which is why I'm being so insistent.

I do think the suggesting below is a decent alternative:

> A cloud service automatically evaluates proposed changes to catch errors and provide continuous publishing.

However, I think it's a bit more jargon-heavy. One goal with the current construct is to not lose readers who don't understand what  "Continuous Publication" means.

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: Done in https://github.com/greenelab/meta-review/pull/211/commits/b95a292b9e7b315cafa57e460717182a98d952f0

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: Done in https://github.com/greenelab/meta-review/pull/211/commits/b95a292b9e7b315cafa57e460717182a98d952f0.

I had put those two closing sentences since I thought there would be a strong desire to emphasize Deep Review and the challenges there. Since we started this manuscript more as a discussion as those challenges and it evolved to be more about Manubot generally. I don't have a strong preference, and prefer the stronger closing statement for the abstract with those 2 sentences gone.

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: One final thing is that we should think about whether to use the term "continuous publishing" or "continuous publication". There are a few instances in the manuscript, and we should make them all consistent.

From https://wikidiff.com/publishing/publication:

> As nouns the difference between publishing and publication is that publishing is the industry of publishing, including the production and distribution of books, magazines, web sites, newspapers, etc while publication is the act of publishing printed or other matter. 

http://blog.martinfenner.org/2014/03/10/continuous-publishing/

Comment (cgreene) [reply content/01.abstract.md]: I like publishing. 👍

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md:132]: Currently, the link is to https://greenelab.github.io/meta-review/v/4b6396bcefd1b9c7ddf39c1d3f0b3eab2dd63f31/. I was actually thinking of going even older, although think the current one is fine. Basically, I think it's more meaningful to show a version that is different. Having fewer authors is kind of cool because it demonstrates the more continuous nature of Manubot manuscripts.

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: continuous publishing has [674 Google Scholar results](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C39&q=%22continuous+publishing%22&btnG=) versus [4,400](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C39&q=%22continuous+publication%22&btnG=) for "continuous publication".

However, much of the usage of "continuous publication" is regarding journals moving away from issue-based publishing. [Example](https://doi.org/10.1259/bjr.20130252)

> Continuous publication focuses on publishing individual articles as opposed to the more traditional issue-based publishing. Once an article is in its final form, after editing and proofreading, it is immediately published online as the version of record straight into an issue. There is no delay caused by waiting for a group of manuscripts to be gathered together for a given issue. This drip feeding of articles means that issues are built up gradually throughout the month.

Continuous publishing is used by Fenner's [post](http://blog.martinfenner.org/2014/03/10/continuous-publishing/) to describe Manubot's use case.

Hence, I feel like we're slightly coining the term, so we should probably just go with what we think is most accurate?

Comment (agitter) [reply content/01.abstract.md]: @dhimmel I'll defer to you regarding the punctuation and the publishing / publication decision.  Whatever you find to be more accurate works for me.

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md:132]: Good point.  As long as it is a deliberately selected version, I'm okay with it.  I do like the idea of showing major changes to the manuscript.

**Review by agitter — APPROVED** [review]
@dhimmel I'm approving because all of my major comments have been addressed.  There are still some minor things to update before merging, like choosing continuous publication / publishing and making it consistent in the rest of the manuscript.

**Comment by dhimmel** [comment]
> There are still some minor things to update before merging, like choosing continuous publication / publishing and making it consistent in the rest of the manuscript.

Can we do this in a subsequent PR. I'd rather merge this and then work from there.

File: content/01.abstract.md
Comment (dhimmel) [reply content/01.abstract.md]: Will address in https://github.com/greenelab/meta-review/pull/216

**Pull request merged**
